code/modules/vehicles/cars/clowncar.dm

```diff
@@ -49,6 +49,24 @@
 /obj/vehicle/sealed/car/clowncar/mob_forced_enter(mob/M, silent = FALSE)
 	. = ..()
 	playsound(src, pick('sound/vehicles/clowncar_load1.ogg', 'sound/vehicles/clowncar_load2.ogg'), 75)
+	if(iscarbon(M))
+		var/mob/living/carbon/forced_mob = M
+		if(forced_mob.has_reagent(/datum/reagent/consumable/ethanol/irishcarbomb))
+			var/reagent_amount = forced_mob.reagents.get_reagent_amount(/datum/reagent/consumable/ethanol/irishcarbomb)
+			forced_mob.reagents.del_reagent(/datum/reagent/consumable/ethanol/irishcarbomb)
+			var/obj/item/organ/stomach/stomach = forced_mob.getorganslot(ORGAN_SLOT_STOMACH)
+			if(stomach)
+				reagent_amount += stomach.reagents.get_reagent_amount(/datum/reagent/consumable/ethanol/irishcarbomb)
+				stomach.reagents.del_reagent(/datum/reagent/consumable/ethanol/irishcarbomb)
```

Comment:
Do you need all this? I think I'd rather you just `get_reagent_amount` from `forced_mob.reagents`, like everything else does, or split it into its own proc of "total" reagent amount or something.

Reply:
It's not necessary, and should just ```get_reagent_amount``` like everything else.